opal/runtime/opal_params.c

```diff
@@ -275,6 +276,9 @@ int opal_register_params(void)
         return ret; 
     }
 
+    /* register the common_verbs parameters */
```

Comment:
This breaks all kinds of abstractions.  :-(

This should not be here -- you cannot call common components from the core.  "Common" components are only intended to be shared code between components (usually in different frameworks).